prefect_saturn/core.py

```diff
@@ -126,7 +125,7 @@ def _hash_flow(self, flow: Flow) -> str:
             tenant_id,
         ]
         hasher = hashlib.sha256()
-        hasher.update(cloudpickle.dumps(identifying_content))
+        hasher.update(json.dumps(identifying_content).encode("utf-8"))
```

Comment:
Another option would be to hardcode the pickle protocol right?

Reply:
hm, I'm not sure if that gives us the same guarantees, but that does seem to work in my test case. The advantage of hardcoding the pickle protocol is that we could maintain compatibility for existing users.

Reply:
yeah dask hardcodes the pickle protocol I think.

Reply:
@jsignell do you know if dask uses pickle/cloudpickle in hashing functions?

Reply:
yeah dask does things like: 

https://github.com/dask/dask/blob/61db08d29d2f18149c912249198ceee152d84857/dask/base.py#L980-L992